type.go

```diff
@@ -106,7 +106,9 @@ func (t Type) GoType() string {
 		// an integer type.
 		return t.pointers() + "uintptr"
 	case "GLsync":
-		return t.pointers() + "unsafe.Pointer"
+		// GLsync is treated as an opaque, pointer-width type. Additional special
+		// case handling is required for the corresponding typedef, see below.
```

Comment:
It's not very clear to me what "below" refers to. Do you mean `CTypedef`? Then it'd be better to say "see CTypedef."

Reply:
Done.